condor/condor_meter

```diff
@@ -551,11 +551,14 @@ def determine_host_description(classad):
 
     If there's no special host description, this returns None.
     """
-    if 'MATCH_EXP_JOBGLIDEIN_ResourceName' not in classad:
+    try:
+        if classad['MATCH_EXP_JOBGLIDEIN_ResourceName'] == 'Local Job':
+            host_descr = GratiaCore.Config.get_SiteName()
+        else:
+            host_descr = classad['MATCH_EXP_JOBGLIDEIN_ResourceName']
+    except KeyError:
```

Comment:
@brianhlin, how would you feel about just using an if/elif/else here?  (something like [here](https://github.com/opensciencegrid/gratia-probe/blob/master/condor/condor_meter#L745))

I'm not really wild about the try/except thing (does a `KeyError` only come from `classad[]` or can `get_SiteName()` generate one too?) ... But if you're really feeling the try/except thing that works too.